frontend/src/components/ElectionForm/TimeZones.ts

```diff
@@ -0,0 +1,137 @@
+export const timeZones = [
+    { ID: 'Pacific/Midway', name: 'Midway Island, Samoa' },
```

Comment:
This list could get hard to managed over time

I'm surpised Luxon doesn't have an API for getting a list of timezones, but there are some workarounds listed here https://github.com/moment/luxon/issues/353

Reply:
I read through that thread a bit before. The issue is that I don't want to have a list of ALL timezones, just the most common ones that will allow 99.9% of users to find one so users don't have to scroll through 300 time zones. I should also note that the default time zone the browser uses will be selected first and doesn't have to be in that list.

This list is what Zoom uses, so I'm fairly comfortable using it. But getting a managed library would probably be better.

Reply:
Sounds good, I was worried we'd be signing up to keep this in sync as new timezones are modified or added but if it's a subset then that's easier

Reply:
Yea definitely just a subset